web/src/__tests__/server/api/otel/otelMapping.servertest.ts

```diff
@@ -0,0 +1,303 @@
+                unsigned: true,
+              },
+              endTimeUnixNano: {
+                low: 1327836088,
+                high: 404677085,
+                unsigned: true,
+              },
+              attributes: [
+                {
+                  key: "telemetry.sdk.name",
+                  value: { stringValue: "openlit" },
+                },
+                { key: "gen_ai.system", value: { stringValue: "openai" } },
+                {
+                  key: "gen_ai.operation.name",
+                  value: { stringValue: "chat" },
+                },
+                {
+                  key: "gen_ai.endpoint",
+                  value: { stringValue: "openai.chat.completions" },
+                },
+                {
+                  key: "gen_ai.response.id",
+                  value: {
+                    stringValue: "chatcmpl-AugxBIoQzz2zFMWFoiyS3Vmm1OuQI",
+                  },
+                },
+                {
+                  key: "gen_ai.environment",
+                  value: { stringValue: "default" },
+                },
+                {
+                  key: "gen_ai.application_name",
+                  value: { stringValue: "default" },
+                },
+                {
+                  key: "gen_ai.request.model",
+                  value: { stringValue: "gpt-3.5-turbo" },
+                },
+                {
+                  key: "gen_ai.request.top_p",
+                  value: { doubleValue: 1 },
+                },
+                {
+                  key: "gen_ai.request.max_tokens",
+                  value: { intValue: { low: -1, high: -1, unsigned: false } },
+                },
+                {
+                  key: "gen_ai.request.user",
+                  value: { stringValue: "" },
+                },
+                {
+                  key: "gen_ai.request.temperature",
+                  value: { doubleValue: 1 },
+                },
+                {
+                  key: "gen_ai.request.presence_penalty",
+                  value: { doubleValue: 0 },
+                },
+                {
+                  key: "gen_ai.request.frequency_penalty",
+                  value: { doubleValue: 0 },
+                },
+                { key: "gen_ai.request.seed", value: { stringValue: "" } },
+                {
+                  key: "gen_ai.request.is_stream",
+                  value: { boolValue: false },
+                },
+                {
+                  key: "gen_ai.usage.input_tokens",
+                  value: { intValue: { low: 14, high: 0, unsigned: false } },
+                },
+                {
+                  key: "gen_ai.usage.output_tokens",
+                  value: { intValue: { low: 96, high: 0, unsigned: false } },
+                },
+                {
+                  key: "gen_ai.usage.total_tokens",
+                  value: { intValue: { low: 110, high: 0, unsigned: false } },
+                },
```

Comment:
input_tokens, output_tokens, total_tokens map to input, output, total. For all others we keep the key from the original. Same for prompt_tokens, completion_tokens.